signals.go

```diff
@@ -112,9 +112,8 @@ func (h *signalHandler) forward(process *libcontainer.Process, tty *tty, detach
 			// and it should not be forwarded to the container.
 			// Do nothing.
 		default:
-			us := s.(unix.Signal)
```

Comment:
Maybe we need to discuss this behavior change here:
We didn't check this transform result in before, if failed, we will also send a null signal to the container.
But after this refactor, we will get an error msg here.

1) Maybe the failure possibility of this transform is very small;
or
2) Maybe this is a bug in before?

Reply:
Please see the implementation in `exec_unix.go`:
```
func (p *Process) signal(sig Signal) error {
	s, ok := sig.(syscall.Signal)
	if !ok {
		return errors.New("os: unsupported signal type")
	}
```